### ISSUE TITLE
Perform accessibility enhancements

### ISSUE THREAD

**Issue by talitapagani** [wejs/site#4]
According to the Accessibility Evaluation conducted at We.js website, taking into account WCAG 2.0 guidelines, the following adjustments need to be performed:
## WCAG 2.0 Level A
- [ ] Add `lang='en'` attribute to the `<html>` tag. It's needed to inform to screen readers the natural language of the website content;
- [ ] Add and make revision of alternative (ALT) text in images. The ALT text should be relevant, concise and represent the image content or meaning that is relevant to the context;
- [ ] Add missing attribute `for` to form `<label>`, specially at Login and Register pages;
- [ ] Add a unique `<title>` for each We.js page, following W3C/WAI guidelines;
- [ ] Add a _skip to content_ link at the beginning of pages. This link skip the navigation menu and header, jumping to the main content;
- [ ] Add ARIA landmark roles for header, search, menu, footer and aside content.
## WCAG 2.0 Level AA
- [ ] Add heading titles (`<h1>`) for pages that are missing them, such as Plugins and Themes;
- [ ] Make focus visible for form fields;
- [ ] Improve error messages, specially on form pages;
- [ ] Add ARIA roles, states and properties to page elements;
- [ ] Improve color contrast between elements background and foreground colors;
- [ ] Provide a high contrast mode.

**Comment by albertosouza** [comment]
Hi @talitapagani i think is better to add this issue in main we.js repo, link: https://github.com/wejs/we
This updates is related to all we.js themes and projects then we can work in main repo
